### DISCUSSION TITLE
The use of page-top.md or what's the simplest directory layout for just a few pages

### DISCUSSION THREAD

**Discussion by myshkin5** [McShelby/hugo-theme-relearn#433] [Q&A] [answered]
Hello!

I throwing together a handful of pages as a starting point. I'm thinking that my left nav/menu will just list around a half dozen pages. One of the pages has supporting images so I put it in a separate directory and it's showing up in the menu fine. Other pages are more simple so I put them next to my top-level `_index.md` but they aren't showing up in the menu. So borrowing from [the docs](https://mcshelby.github.io/hugo-theme-relearn/cont/pages/index.html#folders), my content looks something like this:

````plaintext
content
├── level-one
│   ├── _index.md                   <-- /level-one
│   ├── x.svg                       <-- /level-one/x.svg
│   ├── y.svg                       <-- /level-one/y.svg
│   ├── z.svg                       <-- /level-one/z.svg
│   └── page-1-c.md                 <-- /level-one/page-1-c
├── _index.md                       <-- /
└── page-top.md                     <-- /page-top
````

Is there a better layout recommended for starting out with a few simple pages? Is there some front matter setting to get `page-top.md` to show up in the menu? Or do all of my level-one pages need to be in their own directory? Also if I'm missing the point of `page-top.md`, please explain it (sorry, I'm a backend engineer so a lot of these concepts are foreign to me).

Thanks in advance!

**Comment by McShelby** [comment, marked as answer]
On the top level there are only sections supported at the moment. So your layout for page-top.md should look like

````
content
├── level-one
│   ├── _index.md                   <-- /level-one
│   ├── x.svg                       <-- /level-one/x.svg
│   ├── y.svg                       <-- /level-one/y.svg
│   ├── z.svg                       <-- /level-one/z.svg
│   └── page-1-c.md                 <-- /level-one/page-1-c
├── _index.md                       <-- /
└── page-top                       
    └── _index.md                   <-- /page-top
````

**Comment by myshkin5** [comment]
Thank you!

**Comment by McShelby** [comment]
Fixed in #434 and pushed to main branch. Will be released in version 5.9.3
